README.md

```diff
@@ -32,68 +79,144 @@ and [test](https://en.wikipedia.org/wiki/Training,_validation,_and_test_sets#Tes
 This terminology has lead to [some confusion in the past](https://en.wikipedia.org/wiki/Training,_validation,_and_test_sets#Confusion_in_terminology). 
 Splitting the `hindcast-input` (`training-input`) dataset between training and validation is standard way and should be decided carefully.
 
-The `forecast-input` (`test-input`) must not be used as a validation dataset  : it must not be used to tune the hyperparameters or make decision about the ML model. 
-Fostering discussions about how to prevent overfitting may be an outcome of the challenge.
```

Comment:
Is it not true ?

Reply:
I didn't like the second sentence. We know what we need to do against overfitting. I will keep the first

Reply:
ok